src/app/Http/Controllers/API/PollController.php

```diff
@@ -38,7 +39,18 @@ public function index(Presentation $presentation)
      */
     public function store(Request $request, Presentation $presentation)
     {
-        //
+        $request->validate([
+            'subject' => 'required',
+            'status' => 'required'
+        ]);
+
+        $poll_data = $request->only([
+            'subject',
+        ]);
+        $poll_data['status'] = 'new';
+        $poll = $presentation->polls()->create($poll_data);
```

Comment:
You should check if the user as the rights to create a new poll

Reply:
Fixed in da150f9